src/labelle/lib/devices/dymo_labeler.py

```diff
@@ -240,23 +241,81 @@ class DymoLabeler:
 
     LABELER_DISTANCE_BETWEEN_PRINT_HEAD_AND_CUTTER_MM = 8.1
     LABELER_PRINT_HEAD_HEIGHT_MM = 8.2
-    SUPPORTED_TAPE_SIZES_MM = (24, 19, 12, 9, 6)
     DEFAULT_TAPE_SIZE_MM = 12
 
+    # Default supported tape sizes
+    labeler_supported_tape_sizes: list
+
+    # Default printer head size in dots
+    labeler_print_head_width_pixels: int
+
+    # Default printer margins (for each tape size)
+    # { tapeSizeMm : (firstVisiblePixel, lastVisiblePixel) }
+    labeler_label_vertical_margins: dict[int, Any]
+
     def __init__(
         self,
         tape_size_mm: int | None = None,
         device: UsbDevice | None = None,
     ):
         if tape_size_mm is None:
             tape_size_mm = self.DEFAULT_TAPE_SIZE_MM
```

Comment:
Is this still relevant? Is `DEFAULT_TAPE_SIZE_MM` supported by all printer models? I guess we should choose the first available tape size for the model instead.

Reply:
I'm not really happy about this either, but the whole software kind of relies on it (also with the GUI)
Maybe I should convert it to take the smallest or largest supported tape size?

Reply:
Fixed it, it now takes the highest supported size if not set